src/pages/LogView/LoadingPage/LoadingBar/index.tsx

```diff
@@ -36,7 +39,31 @@ const Bar = styled.div<{ progress: number }>`
     border-top-right-radius: ${size.xxs};
     border-bottom-right-radius: ${size.xxs};
     `}
-  background: ${green.base};
+  height: 6px;
+  background-color: ${green.base};
+  ${({ indeterminate, progress }) =>
+    indeterminate
+      ? indeterminateAnimation
+      : `  width: ${progress}%;
+`}
   box-shadow: 0 0 ${size.xs} ${green.light2};
 `;
+
+const indeterminateAnimation = css`
+  position: relative;
+  bottom: 0;
+  top: 0;
+  width: 50%;
+
+  /* Move the bar infinitely */
+  animation: indeterminate-progress-bar 2s infinite;
```

Comment:
nit. probably not that important and not sure if it's fixable, but wanted to flag that the animation freezes before the log loads in (while the leaf is still moving properly)

https://user-images.githubusercontent.com/47064971/231023960-9c76b958-828c-4880-aacc-411fe88c442c.mov

Reply:
I can't figure out a way around it it freezes when processing the log which is an expensive operation. This should only happen for resmoke logs.